__tests__/app.test.js

```diff
@@ -17,6 +18,17 @@ afterAll(() => {
   db.end();
 });
 
+describe("GET /api", () => {
+  test("200: responds with an object including details of all available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.endpoints).toEqual(endpoints);
+      });
+  });
```

Comment:
Really nice neat solution here. Often students can overcomplicate this ticket, well done.